src/styles/typography.css

```
@@ -51,9 +55,9 @@ body {
 /* --------------------------------------------------------------------------
    iPad Typography Tokens
    -------------------------------------------------------------------------- */
-@utility text-ipad-heading-1-medium-36 {
+@utility text-ipad-heading-0-medium {
```

Stylelint (17.14.0)

[error] Unexpected unknown at-rule "`@utility`" (scss/at-rule-no-unknown)

(scss/at-rule-no-unknown)